node.js

```diff
@@ -8,30 +16,43 @@ const { Worker } = require("worker_threads");
 const PROTO_PATH = path.resolve(__dirname, "./protos/chord.proto");
 
 const packageDefinition = protoLoader.loadSync(PROTO_PATH, {
-  keepCase: true,
-  longs: String,
-  enums: String,
-  defaults: true,
-  oneofs: true
+    keepCase: true,
```

Comment:
We've got a bit of noise in the commit due to formatting issues. I added an issue to add Prettier to this repo so that we can consistently have format on save behavior that behaves consistently across our machines.